packages/app/src/app/pages/NewDashboard/Components/Filters/ViewOptions/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FunctionComponent } from 'react';
+import { useOvermind } from 'app/overmind';
+import { Text, Menu, Stack } from '@codesandbox/components';
+import { GridIcon, ListIcon } from './icons';
+
+const STATES: { name: string; key: 'grid' | 'list'; icon: any }[] = [
```

Comment:
```suggestion
const STATES: { name: string; key: 'grid' | 'list'; icon: ComponentType }[] = [
```

Reply:
I don't really see the sue in adding that, it has never helped me really :/

Reply:
Using `any` is an anti-pattern and `ComponentType` is the correct type here